examples/seismic/acoustic/wavesolver.py

```diff
@@ -14,19 +14,28 @@ class AcousticWaveSolver(object):
     and encapsulates the time and space discretization for a given problem
     setup.
 
-    :param model: Physical model with domain parameters
-    :param source: Sparse point symbol providing the injected wave
-    :param receiver: Sparse point symbol describing an array of receivers
-    :param time_order: Order of the time-stepping scheme (default: 2, choices: 2,4)
-                       time_order=4 will not implement a 4th order FD discretization
-                       of the time-derivative as it is unstable. It implements instead
-                       a 4th order accurate wave-equation with only second order
-                       time derivative. Full derivation and explanation of the 4th order
-                       in time can be found at:
-                       http://www.hl107.math.msstate.edu/pdfs/rein/HighANM_final.pdf
-    :param space_order: Order of the spatial stencil discretisation (default: 4)
-
-    Note: space_order must always be greater than time_order
+    Parameters
+    ----------
+    model : Model
+        Physical model with domain parameters.
+    source : PointData
+        The injected wave.
+    receiver : PointData
+        An array of receivers.
+    time_order : int, optional
+        Order of the time-stepping scheme (default: 2, choices: 2,4)
+        time_order=4 will not implement a 4th order FD discretization
+        of the time-derivative as it is unstable. It implements instead
+        a 4th order accurate wave-equation with only second order
+        time derivative. Full derivation and explanation of the 4th order
+        in time can be found at:
+        http://www.hl107.math.msstate.edu/pdfs/rein/HighANM_final.pdf
+    space_order: int, optional
+        Order of the spatial stencil discretisation. Defaults to 4.
+
+    Notes
+    -----
+    Space_order must always be greater than time_order.
```

Comment:
\``space_order\`` must always be greater than \``time_order\``.

But I'm confused (again). Time order doesn't even appear in the parameters list...so this is just a leftover that needs to be dropped? @mloubout

Reply:
No this is not true to be removed, again I was sure I removed it but obviously didn't.